app/src/main/java/com/example/weathertimeapp/mvp/presenter/WeatherTimePresenter.kt

```diff
@@ -8,10 +8,29 @@ class WeatherTimePresenter(
     private val view: WeatherTimeContracts.View
 ) : WeatherTimeContracts.Presenter {
 
+    override fun initPresenter() {
+        createAndSetCitiesList()
+        setSoftInputMode()
+        initRecyclerView()
+        createAndShowData()
+    }
+
     override fun createAndSetCitiesList() {
```

Comment:
Is this called from other class? if not, please make it private

Reply:
done, thanks